suite-native/module-settings/src/components/TurnOffCheckScreenContent.tsx

```diff
@@ -85,21 +87,29 @@ export const TurnOffFirmwareAuthenticityCheckModalScreen = () => {
             <VStack spacing="sp32" flex={1}>
```

Comment:
There seems to be some top margin missing:
<img width="532" alt="image" src="https://github.com/user-attachments/assets/f23aee4d-5dae-4035-9841-08e50d498c5f" />

Reply:
[`cbbd7ea` (#18921)](https://github.com/trezor/trezor-suite/pull/18921/commits/cbbd7ea8a83fccb9a577c53936ad2dc72c812fa3)